README.md

```diff
@@ -1,23 +1,37 @@
 # OpenFisca Core
 
-[![Build Status](https://travis-ci.org/openfisca/openfisca-core.svg?branch=master)](https://travis-ci.org/openfisca/openfisca-core)
+[OpenFisca](https://doc.openfisca.fr/) is a versatile microsimulation free software. Check the [online documentation](https://doc.openfisca.fr/) for more details.
 
-[More build statuses](https://www.openfisca.fr/build-status)
+This package contains the core features of OpenFisca, which are meant to be used by country packages such as [OpenFisca-France](https://github.com/openfisca/openfisca-france). Bootstrapping your own country package should not take more than 5 minutes: check our [country package template](https://github.com/openfisca/country-template).
 
-[OpenFisca](https://www.openfisca.fr/) is a versatile microsimulation free software.
-This is the source code of the Core module.
+## Environment
 
-The documentation of the project is hosted at https://doc.openfisca.fr/
+This package requires Python 2.7
 
-## Install for development
+## Installation
 
-This is the way to install OpenFisca-Core for development. If you just want to modify the legislation formulas and/or parameters, just follow the [installation section](https://doc.openfisca.fr/install.html) of the documentation.
+If you're developping your own country package, you don't need to explicitely install OpenFisca-Core. It just needs to appear [in your package dependencies](https://github.com/openfisca/openfisca-france/blob/18.2.1/setup.py#L53).
 
-This package requires Python 2.7.
+If you want to contribute to OpenFisca-Core itself, welcome! To install it locally in developpment mode :
 
 ```bash
 git clone https://github.com/openfisca/openfisca-core.git
 cd openfisca-core
-pip install --editable .
-python setup.py compile_catalog
+pip install --editable ".[test]"
 ```
+
+## Testing
+
+```sh
+make test
+```
+
+## Serving the API
+
+OpenFisca-Core provides a Web-API. To run it with the mock country package `openfisca_dummy_country`, run:
```

Comment:
Suggestions are welcome if you think it's worth expliciting where the package comes from.